schema1.sql

```diff
@@ -0,0 +1,14 @@
+-- An user has many blog posts
+
+create table users (
+  id serial primary key,
+  name text
+);
+
+create table posts (
+  id serial primary key,
+  title text,
+  date timestamp,
```

Comment:
You may need to name this field something like `post_date`, as `date` is a reserved word in sql: https://www.postgresql.org/docs/current/static/datatype-datetime.html